Build/Build.cs

```diff
@@ -287,7 +287,12 @@ class Build : NukeBuild
         });
 
     bool HasDocumentationChanges =>
-        Changes.Any(x => x.StartsWith("docs"));
+        Changes.Any(x => 
```

Comment:
🤔 Shouldn't we also trigger this if anything in `_build` folder changes?

Reply:
On documentation changes?

Reply:
Where is the `_build` folder btw?

Reply:
The folder is called `Build`. The project is called `build.csproj`

If something changes related to how to do spell checks or we update any dependencies related to that, you also want to run the full build. Right?

Reply:
Sure.. I've forgotten that either :D